src/parser/github-comment-module.ts

```diff
@@ -153,6 +153,10 @@ export class GithubCommentModule extends BaseModule {
       content.push(buildContributionRow("Issue", "Task", result.task.multiplier, result.task.reward));
     }
 
+    if (result.simplificationReward && result.simplificationReward.reward != 0) {
```

Comment:
```suggestion
    if (result.simplificationReward && result.simplificationReward.reward !== 0) {
```

Avoid type coercions